### PR TITLE
add archer api file upload and job submit scripts

### PR DESCRIPTION
**Pull request by wintun12**
- add main.sh that upload the files and run the job on Archer
- add Dockerfile to create docker image
- add README file for the info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,35 @@
-# archer_api_upload
\ No newline at end of file
+# archer_api_upload
```
Comment (natashapinto) [inline README.md]: Could add the version number in the heading please

File: main.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```
Comment (natashapinto) [inline main.sh:1]: any reason why the script is in bash?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:latest
+LABEL maintainer="seglh"
```
Comment (natashapinto) [inline Dockerfile:2]: has the docker image been pushed to the seglh dockerhub?

**Review by natashapinto — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (wintun12) [reply README.md]: Added the version in the heading as advised.

File: Dockerfile (hunk 3 above)
Comment (wintun12) [reply Dockerfile:2]: Not yet. If you are happy with the script, I can push it to seglh dockerhub.

File: main.sh (hunk 2 above)
Comment (wintun12) [reply main.sh:1]: yes, the command lines (both for file uploading and job submission) recommended by Archer API documentation are in bash. Also the sample list preparation for the job submission is quite convenient in bash.

File: Dockerfile (hunk 3 above)
Comment (natashapinto) [reply Dockerfile:2]: great, thanks

**Pull request merged**